src/game.cpp

```diff
@@ -27,7 +27,9 @@ int Game::run() {
 }
 
 void Game::generate_content() {
-    StochasticBinarySpacePartitioningLevelGenerator level_generator(BoundingBox2f::from_zero(640.0, 480.0));
+    constexpr int GRID_WIDTH = BOARD_WIDTH / Grid::ELEMENT_SIZE;
+    constexpr int GRID_HEIGHT = BOARD_HEIGHT / Grid::ELEMENT_SIZE;
```

Comment:
I suggest to extract GRID_WIDTH and GRID_HEIGHT constants the same way you did with WINDOW_*. Just remove the logic with Grid::ELEMENT_SIZE. I understand, that this logic prevents renderer from being unable to display whole grid, but as I stated in this review, Grid class should not care about how many pixels one cell is rendered as

Reply:
I will just create bbox with 128 width and 96 height, I cannot fully understand what do you want me to do since grid already has grid_* constants. You are free to change it in any way you will like later